src/components/Venues/Modal/VenueModal.tsx

```
@@ -293,27 +289,41 @@
+                } catch (error) {
+                  toast.error('Failed to upload image');
+                }
               }
             }}
             className={styles.inputField}
           />
-          {venueImage && (
+          {imagePreviewUrl && (
             <div className={styles.previewVenueModal}>
-              <img src={imageURL} alt="Venue Image Preview" />
+              <img src={imagePreviewUrl} alt="Venue Image Preview" />
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)